scania-digital-design-system/sdds
# feat(colour): migrate colours story

**Describe pull-request**  
Migrate the colours stories and add controls to be able to flip through colours on scale. 

**Solving issue**  
Fixes: [AB#2477](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2477)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in Foundations -> Colours
3. Make sure the correct colors and scales are represented. 

**Screenshots**  
-

**Additional context**  
-
